test/viewmodels/SummaryCardHelperSpec.scala

```diff
@@ -364,70 +361,70 @@ class SummaryCardHelperSpec extends SpecBase with ImplicitDateFormatter {
     "return SummaryCards" when {
       "given a Penalty point" when {
         "populateLateSubmissionPenaltyCard is called" in {
-          val result = helper.populateLateSubmissionPenaltyCard(Seq(sampleReturnSubmittedPenaltyPoint), quarterlyThreshold, quarterlyThreshold - 1)
+          val result = helper.populateLateSubmissionPenaltyCard(Seq(sampleLateSubmissionPointReturnSubmitted), quarterlyThreshold, quarterlyThreshold - 1)
           result shouldBe Seq(sampleLSPSummaryCardReturnSubmitted)
         }
 
         "user has removed points below active points - active points should be reindexed so that the points are logically numbered correctly" in {
           val sample3ReturnsSubmittedPenaltyPointDataAndOneRemovedPointv2: Seq[LSPDetails] = Seq(
-            samplePenaltyPoint.copy(penaltyOrder = "4"),
-            samplePenaltyPoint.copy(penaltyOrder = "3"),
-            samplePenaltyPoint.copy(penaltyOrder = "2"),
+            sampleLateSubmissionPoint.copy(penaltyOrder = "4"),
+            sampleLateSubmissionPoint.copy(penaltyOrder = "3"),
+            sampleLateSubmissionPoint.copy(penaltyOrder = "2"),
             sampleRemovedPenaltyPoint
           )
           val expectedResult: Seq[LateSubmissionPenaltySummaryCard] = Seq(LateSubmissionPenaltySummaryCard(
             Seq(
               helper.summaryListRow(
                 period,
-                Html(vatPeriodValue(dateToString(sampleDate), dateToString(sampleDate)))
+                Html(vatPeriodValue(dateToString(sampleDateLSP), dateToString(sampleDateLSP)))
               ),
-              helper.summaryListRow(returnDue, Html(dateToString(sampleDate))),
-              helper.summaryListRow(returnSubmitted, Html(dateToString(sampleDate))),
+              helper.summaryListRow(returnDue, Html(dateToString(sampleDateLSP))),
+              helper.summaryListRow(returnSubmitted, Html(dateToString(sampleDateLSP))),
```

Comment:
Is it possible to use taxPeriodStart, taxPeriodEnd, taxPeriodDue and receiptDate here?